projects/plugins/super-cache/wp-cache.php

```diff
@@ -3214,10 +3212,16 @@ function wp_cron_preload_cache() {
 		$permalink_counter_msg = $cache_path . "preload_permalink.txt";
 		if ( isset( $wp_cache_preload_taxonomies ) && $wp_cache_preload_taxonomies ) {
 			$taxonomies = apply_filters( 'wp_cache_preload_taxonomies', array( 'post_tag' => 'tag', 'category' => 'category' ) );
+
+			$preload_more_taxonomies = false;
+
 			foreach( $taxonomies as $taxonomy => $path ) {
 				$taxonomy_filename = $cache_path . "taxonomy_" . $taxonomy . ".txt";
-				if ( $c == 0 )
+
+				if ( ! get_transient( 'taxonomy_preload' ) ) {
 					@unlink( $taxonomy_filename );
+				}
+				set_transient( 'taxonomy_preload', 1, 600 );
```

Comment:
I think it would be worth adding a comment explaining the purpose of `'taxonomy_preloead'` to make this code easier to understand on future reads.

Reply:
Done!